kuzzleio/documentation-V2
# Run one test

## What does this PR do?

This PR adds the ability to run just one test by specifying the path of the `.yml` file that describes it. This is handy while writing a feature, to ensure that the code snippet is correct without having to test the whole documentation.

### How should this be manually tested?

  - Clone the repo and cd into it
  - `sh run-test.sh -l js -o index/code-example/create.yml`
  - check that only one test is performed

### Other changes

While adding this feature, I also added an option to enable running the tests using a running Kuzzle stack (the `-n` option). This is handy when testing locally and launching the tests many times: not having to run and kill the whole Kuzzle stack saves time.

### Boyscout

Some typos in the README have been fixed.
